src/backend/base/langflow/components/nvidia/nvidia_ingest.py

```
@@ -162,6 +176,14 @@
             self.log(err_msg)
             raise ValueError(err_msg)
 
+        # Check if all files are PDFs when high resolution mode is enabled
+        if self.high_resolution:
+            non_pdf_files = [file for file in file_list if not str(file.path).lower().endswith('.pdf')]
```

Ruff (0.11.9)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)